downstream/modules/builder/con-system-dependencies.adoc

```diff
@@ -2,7 +2,7 @@
 
 = System
 
-The `system` entry in the definition points to a https://docs.opendev.org/opendev/bindep/latest/readme.html[bindep] requirements file or to an inline list of bindep entries, which will install system-level dependencies that are outside of what the collections already include as their dependencies. It can be listed as a relative path from the directory of the {ExecEnvNameSing} definition’s folder, or an absolute path. A minimum expectation is that the collection(s) specify necessary requirements for `[platform:rpm]`.
+The `system` entry in the definition points to a link:https://docs.opendev.org/opendev/bindep/latest/readme.html[bindep] requirements file or to an inline list of bindep entries, which install system-level dependencies that are outside of what the collections already include as their dependencies. It can be listed as a relative path from the directory of the {ExecEnvNameSing} definition's folder, or as an absolute path. At a minimum, the the collection(s) should specify necessary requirements for `[platform:rpm]`.
```

Comment:
Suggest changing "should specify necessary" to "must specify necessary". "Should" makes the requirements sound optional and I don't think they are.

Reply:
I was wondering about that. Thanks for the clarification!